### PR TITLE
BC-1643 - add school specific privacy policy

### PR DESCRIPTION
**Pull request by davwas**
# Description

## Links to Tickets or other pull requests
https://ticketsystem.dbildungscloud.de/browse/BC-1643

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: ansible/group_vars/default/instance_cfg.yml [hunk 1]
```diff
@@ -74,3 +74,6 @@ COLUMN_BOARD_FEEDBACK_LINK: https://easy-feedback.de/umfrage/1683495/cWgx4T
 
 # Ingress
 group_ingress:
+
+# School policy
+FEATURE_SCHOOL_POLICY_ENABLED: "true"
```
Comment (virgilchiriac) [inline ansible/group_vars/default/instance_cfg.yml:79]: if default is already true, no need to overwrite it here

File: ansible/group_vars/nbc/instance_cfg.yml [hunk 2]
```diff
@@ -102,3 +101,6 @@ group_ingress:
   api_v1_registration:
     serviceName: default-backend-404-svc
     servicePort: 8080
+
+# School policy
+FEATURE_SCHOOL_POLICY_ENABLED: "true"
```
Comment (virgilchiriac) [inline ansible/group_vars/nbc/instance_cfg.yml:106]: if default is already true, no need to overwrite it here

**Review by virgilchiriac — CHANGES_REQUESTED** [review]

File: ansible/group_vars/brb/instance_cfg.yml [hunk 3]
```diff
@@ -106,3 +106,6 @@ group_ingress:
     path: /api/v3/fwu
     serviceName: api-fwu-svc
     servicePort: "{{ PORT_FWU_LEARNING_CONTENTS }}"
+
+# School policy
+FEATURE_SCHOOL_POLICY_ENABLED: "true"
```
Comment (virgilchiriac) [inline ansible/group_vars/brb/instance_cfg.yml:111]: also here

**Review by virgilchiriac — CHANGES_REQUESTED** [review]

**Comment by davwas** [comment]
I am closing this PR as it was only used for testing.

**Pull request closed without merging**